resources/definitions/os_discovery/aaron.yaml

```diff
@@ -0,0 +1,123 @@
+mib: INOVONICS-MODEL640-MIB:INOVONICS-MODEL650-MIB
+modules:
+  os:
+    hardware: SNMPv2-MIB::sysName.0
+    serial: INOVONICS-MODEL640-MIB::serial.0
+    version:
+      - INOVONICS-MODEL640-MIB::version.0
+      - INOVONICS-MODEL650-MIB::version.0
+#    version_template: '{{ INOVONICS-MODEL640-MIB::serial.0 }}{{ INOVONICS-MODEL650-MIB::serial.0 }}'
+  sensors:
+    pre-cache:
```

Comment:
Pre-cache should be avoid if possible, it has a negative affect on performance.
But I can't tell if you need it or not because you didn't include MIBs in your PR yet.